sky/backends/backend_utils.py

```diff
@@ -2265,12 +2212,27 @@ def _refresh_cluster(cluster_name):
     return kept_records
 
 
+@typing.overload
+def get_backend_from_handle(
+    handle: 'cloud_vm_ray_backend.CloudVmRayBackend.ResourceHandle'
+) -> 'cloud_vm_ray_backend.CloudVmRayBackend':
+    ...
+
+
+@typing.overload
```

Comment:
Since the argument is a `ResourceHandle` and the return value is a `Backend`, the following does not work:
```
T = TypeVar('T', local_docker_backend.LocalDockerBackend.ResourceHandle, cloud_vm_ray_backend.CloudVmRayBackend.ResourceHandle)

def get_backend_from_handle(handle: T.ResourceHandle) -> T:
```
I could not find a way to get rid of the typing overload. Wdyt?

Reply:
Ahh, I missed that the return type is different..  thanks!